src/window.rs

```diff
@@ -701,6 +701,17 @@ impl Window {
         self.window.is_maximized()
     }
 
+    /// Gets the window's current decorations state.
+    ///
```

Comment:
Could you document use-cases, and relation to `set_decorations`?

Reply:
Why is documenting a use-case necessary? it is simply a getter of a state, and again I just followed the convention of `is_maximized`.

Reply:
Moving them close together is enough for me 👍